docs/styles/spec/file-input-spec.less

```diff
@@ -1,5 +1,5 @@
-@import (reference) 'components/buttons';
-@import (reference) 'components/files/hx-file-control';
+@import (reference) 'components/button/index';
+@import (reference) 'components/file/hx-file-control';
```

Comment:
(nit) we should be able to use `components/file/index` here

Reply:
Sounds good...change made.